docs/content/using-npm/dependency-selectors.md

```diff
@@ -58,10 +58,31 @@ The [`npm query`](/commands/npm-query) commmand exposes a new dependency selecto
 - `:extraneous` when a dependency exists but is not defined as a dependency of any node
 - `:invalid` when a dependency version is out of its ancestors specified range
 - `:missing` when a dependency is not found on disk
-- `:semver(<spec>)` matching a valid [`node-semver`](https://github.com/npm/node-semver) spec
+- `:semver([spec], [selector], [function])` match a valid [`node-semver`](https://github.com/npm/node-semver) version or range to an optional selector (defaults to `[version]`) and providing an optional semver function name
 - `:path(<path>)` [glob](https://www.npmjs.com/package/glob) matching based on dependencies path relative to the project
 - `:type(<type>)` [based on currently recognized types](https://github.com/npm/npm-package-arg#result-object)
 
+##### `:semver([spec], [selector], [function])`
+
+The `:semver()` pseudo selector allows comparing fields from each node's `package.json` using [semver](https://github.com/npm/node-semver#readme) methods. It accepts up to 3 parameters, all of which are optional.
+
+- `spec` a semver version or range (defaults to `*`)
+- `selector` an attribute selector for each node
+- `function` a semver method to apply, one of: `satisfies`, `intersects`, `subset`, `gt`, `gte`, `gtr`, `lt`, `lte`, `ltr`, `eq` or `neq`
+
+If no `spec` is provided this selector will return every node that defines a `version`.
+
+If only a `spec` is provided `selector` defaults to `[version]` and `function` to `satisfies`.
```

Comment:
for the selector default this makes sense for sure, but for the function i'm not sure that it does since the default function varies based on your spec and your selector. that being the case, i can move the selector default to the list above and change this sentence so it only discusses the `function` default